internal/pkg/xpert.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2023 IOTech Ltd
+
+package pkg
+
+import "github.com/edgexfoundry/go-mod-messaging/v2/pkg/types"
+
+func (n NoopClient) PublishBinaryData(data []byte, topic string) error {
+	panic("implement me")
+}
+
+func (n NoopClient) SubscribeBinaryData(topics []types.TopicChannel, messageErrors chan error) error {
+	panic("implement me")
```

Comment:
return the error as the zeromq one
`return fmt.Errorf("not supported SubscribeBinaryData func")`

Reply:
Updated.